docker/lr-transcript_utils/Dockerfile

```diff
@@ -0,0 +1,150 @@
+# Start with a good base python3 image:
```

Comment:
Again, I have the same strong opinion about this whole directory: it deserves its own repo.

Reply:
I've added issue #350 to track this.